pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
not sure switching to using package vars for client setup will be better we get to populating from discovery... for the client defaults, I sort of like having one spot where we retrieve group info